src/igvfd/schemas/sequence_file.json

```diff
@@ -160,6 +160,12 @@
                 "I1",
                 "I2"
             ]
+        },
+        "seqspec": {
+            "title": "Link to seqspec YAML File",
+            "description": "The link to the associated seqspec YAML ConfigurationFile object.",
```

Comment:
please change description to be: The link to the associated seqspec YAML configuration file."

Reply:
changed